notebook.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.            plt.imshow(float('Nan')*Obs[t,0,:])
</pre><p>Instead of <code>float('Nan')</code> I would use the more widely adopted <code>None</code> or <code>np.nan</code>though that may just be my style preference </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/reproduce-deep-prior-4Dvar/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Great suggestions. It's now declared as <code>np.nan</code>. </p>

Reply:
<p>closing conversation, resolved</p>